stability-checker/local/input/testing-common.sh

```diff
@@ -0,0 +1,233 @@
+ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
this is not taken into account when we run our stability on CI pipelines, right? maybe we should just trop that, and add info that you should provide your own testing.sh script or copy it from the kyma if u are interested in executing the same test. WDYT?

Reply:
I think we should have testing script updated to the newest kyma, the earlier script was based on the `helm test` command which we are not using anymore and it's better for me to have it updated. We still have `testing-dummy` script to test the standalone stability checker.

I will just align the `local_helm_install` script to use `testing-dummy`, so this https://github.com/kyma-project/test-infra/tree/master/stability-checker#development will still be working.

Reply:
ok, I deleted those two files :)